person_capture/gui_app.py

```
@@ -7271,6 +8148,28 @@ def _on_ui_change(self, *_args) -> None:
         finally:
             self._in_ui_change = False
 
+    def _clear_prescan_cache(self) -> None:
+        root = _REPO_ROOT / "prescan_cache"
+        try:
+            # Respect a custom relative/absolute cache dir from the current config.
+            raw = str(getattr(self.cfg, "prescan_cache_dir", "prescan_cache") or "prescan_cache").strip()
+            root = Path(raw)
+            if not root.is_absolute():
+                root = _REPO_ROOT / root
+        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)